xdsl/tools/command_line_tool.py

```diff
@@ -21,6 +21,7 @@
 from xdsl.dialects.experimental.fir import FIR
 from xdsl.dialects.experimental.hls import HLS
 from xdsl.dialects.experimental.math import Math
+from xdsl.dialects.fsm import FSM
```

Comment:
wondering whether it makes sense to make it like this for clarity, but not sure if necessary:
```suggestion
from xdsl.dialects.circt.fsm import FSM
```

Reply:
Good point, avoids moving/reorganizing things later.